### ISSUE TITLE
Failed to load icon: linkedin

### ISSUE THREAD

**Issue by wylkon** [atisawd/boxicons#26]
Hi! I would like to thank about your amazing work! And I need some help, this week I updated my boxicons to 1.9.2 version and I started to receive an error:

```shell
Failed to load icon: linkedin
Error: 404 Cannot find "/svg/logos/bxl-linkedin.svg" in boxicons@1.9.2
```

I use React.js and My code is like that:

```js
import 'boxicons';

<box-icon type='logo' name='linkedin'></box-icon>
```

I would like to know if I can do something about that.
Thanks!

**Comment by atisawd** [comment]
thanks for pointing this out , looks like i have made a mistake in the naming of the folder

**Comment by atisawd** [comment]
its been solved in 1.9.3 sorry for the inconvenience

**Comment by wylkon** [comment]
Thank you!
